priv/templates/phx.gen.auth/live_auth_test.exs

```diff
@@ -0,0 +1,47 @@
+defmodule <%= inspect live_auth_module %>Test do
+  use <%= inspect context.web_module %>.ChannelCase
+
+  alias <%= inspect context.module %>
+  alias <%= inspect context.web_module %>.<%= inspect Module.concat(schema.web_namespace, schema.alias) %>LiveAuth
+  alias <%= inspect context.web_module %>.Router.Helpers, as: Routes
+  import <%= inspect context.module %>Fixtures
+
+  setup do
+    socket = %Phoenix.LiveView.Socket{
+      assigns: %{flash: %{}},
+      endpoint: <%= inspect context.web_module %>.Endpoint
+    }
```

Comment:
We should not build this by hand, we probably need an API for this. /cc @chrismccord

Reply:
Thanks José. That's the main point really, but I was not able to find helper methods to build the socket, nor assert the flash or the redirection (the current ones require a view).

The option here would be to ask the use for a path that requires authentication, but that way we would have a failing test out of the box. My preference was to let it as a unit test, but we can make changes also.